Author: Vaclav Ovsik <vaclav.ovsik@i.cz>
Description: Fix "--- more ---" & backspace style paging on netscreen devices.
 (Closes: #520899)
Bug-Debian: http://bugs.debian.org/520899

--- a/bin/nrancid.in
+++ b/bin/nrancid.in
@@ -173,6 +173,7 @@
 
     while (<INPUT>) {
 	tr/\015//d;
+	1 while s/[^[:cntrl:]]\cH//g;
 	next if /^\s*$/;
 	last if(/$prompt/);
 	# throw away the pager prompts
@@ -213,6 +214,7 @@
     print STDERR "    In GetConf: $_" if ($debug);
     while (<INPUT>) {
 	tr/\015//d;
+	1 while s/[^[:cntrl:]]\cH//g;
 	next if /^\s*$/;
 	next if /^Total Config.+$/i;
 	last if(/$prompt/);
